components/learn/modules/ROOT/pages/developing-smart-contracts.adoc

```diff
@@ -153,18 +150,16 @@ We specify a Solidity 0.6 solc version in our `truffle-config.js`.
 ```
 --
 
-[.buidler]
+[.hardhat]
 --
-Our `Box.sol` contract uses Solidity 0.6 so we need to first https://buidler.dev/config/#solc-configuration[configure Buidler to use an appropriate solc version].
+Our `Box.sol` contract uses Solidity 0.6 so we need to first https://hardhat.org/config/#solc-configuration[configure Hardhat to use an appropriate solc version].
 
-We specify a Solidity 0.6 solc version in our `buidler.config.js`.
+We specify a Solidity 0.6 solc version in our `hardhat.config.js`.
 
 ```js
-// buidler.config.js
+// hardhat.config.js
 module.exports = {
-  solc: {
-    version: "0.6.12"
-  }
+  solidity: "0.7.1"
```

Comment:
OpenZeppelin Contract uses Solidity 0.6 as the default so will leave as Solidity 0.6 until this changes
```suggestion
  solidity: "0.6.12",
```